src/Components/Order/ClosingOrder.spec.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ClosingOrder from './ClosingOrder'
+import "@testing-library/jest-dom/extend-expect"
+
+
+describe(('ClosingOrder'), ()=>{
+    test(('should render without crashing'), () =>{
+        const div = document.createElement('div')
```

Comment:
revisa que pasa con este nodo
```suggestion
        const div = document.createElement('div')
```